Update readme.md

MinaBasem opened the pull request:
Hello,

A few additions were made including "Professional Certificates", "Python", "Linux", and a few more links were added.

MinaBasem committed:
Update readme.md

ahmedshaaban1999 commented:
Hi Mina,

Thank you for your inputs. I think we can merge the Linux section with the Shell Scripting section since they both are the same thing, so it will be one section in level 2 called "Linux" instead of "Bash Scripting". 

Also for now at least we are limiting the collaboration at just adding resources for the existing sections. So please remove the "Theoretical" and "Certificates" sections.

MinaBasem committed:
Update readme.md

Removed "Theoretical" and "Certificates".
Also removed "Linux" from level 2 and replaced "Bash Scripting" with it, the links in each were merged.

MinaBasem commented:
Hi @ahmedshaaban1999, I've made the changes you mentioned and submitted a new commit.
